metrics/web/lib/common/presentation/metrics_theme/config/color_config.dart

```diff
@@ -24,4 +24,7 @@ class ColorConfig {
   static const Color lightInputColor = Colors.white;
   static const Color darkInputColor = Color(0xFF0d0d0d);
   static const Color inputSecondaryTextColor = Color(0xFF545459);
+  static const Color darkDropdownBorderColor = Color(0xFF878799);
+  static const Color darkDropdownHoverColor = Color(0xFF1d1d20);
+  static const Color darkDropdownHoverBorderColor = Color(0xFF37373f);
```

Comment:
Recently we are trying to move all theme-specific colors to the theme itself - so the suggestion is to move those colors to the Dark theme - and create similar colors for the Light theme. The reasoning is that we'll still have to do it in the future when we'll work on the light theme so in this case (or have 2 colors in the config light and dark - see last sentence why it's probably not a good idea), it would make much sense to move it to the specific theme. Also changing updating light theme should not affect Dark theme files (or there shouldn't be 2 reasons for the change of Color config).